app/src/main/java/com/codesoom/assignment/repository/TaskRepository.java

```diff
@@ -0,0 +1,65 @@
+package com.codesoom.assignment.repository;
+
+import com.codesoom.assignment.domain.Task;
+import org.springframework.stereotype.Repository;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *  Task 저장소
+ */
+@Repository
+public class TaskRepository {
+
+    private long sequenceId = 0;
+
+    private final Map<Long, Task> tasks = new HashMap<>();
+
+    /**
+     * 주어진 Task 객체를 Task 저장소에 저장합니다.
+     * @param task 저장할 Task 객체
+     */
```

Comment:
이렇게 해보세요.
```suggestion
    /**
     * Task를 저장합니다.
     * @param task 저장할 Task
     */
```
한편 한가지 더 고민할 것이 있습니다. 주석에 `Task`가 그대로 나타나고 있는데요, 이렇게 하면 주석과 Task 클래스의 이름 사이에 커플링이 생깁니다. 즉, Task 클래스 이름을 변경하게 되면 이런 주석들도 전부 찾아서 수정해줘야 하는 문제가 생깁니다.